src/file_list/util.js

```diff
@@ -1,92 +1,91 @@
-import jetpack from 'fs-jetpack'; // module loaded from npm
-import mime from 'mime';
+import jetpack from 'fs-jetpack' // module loaded from npm
+import mime from 'mime'
 
 export default class Util {
-  constructor(fileList) {
-    this.fileList = fileList;
+  constructor (fileList) {
+    this.fileList = fileList
   }
 
   // https://gist.github.com/kethinov/6658166
-  walkSync(dir, filelist) {
-    var files = jetpack.list(dir);
-    filelist = filelist || [];
+  walkSync (dir, filelist) {
+    var files = jetpack.list(dir)
+    filelist = filelist || []
     files.forEach((file) => {
-        if (this.isDir(dir + '/' + file)) {
-            filelist = this.walkSync(dir + '/' + file, filelist);
-        } else {
-            filelist.push(dir + '/' + file);
-        }
-    });
-    return filelist;
+      if (this.isDir(dir + '/' + file)) {
```

Comment:
If you want to use `path.join`, then it should be consistent and used anywhere with path separators.